src/components/SecondaryPanes/XHRBreakpoints.css

```diff
@@ -85,3 +85,21 @@
   position: absolute;
   top: 8px;
 }
+
+.xhr-input-method {
```

Comment:
Whenever we use `-right` or `-left`, it makes us vulnerable to RTL.  We use `margin-inline-start/end` in a few spots, so you'll want to check RTL out.

Reply:
Worked around this by refactoring CSS to have the same left/right values for paddings and margins. 

`margin/padding-inline-start/end` seems like it would have done the trick as well, but was hesitant to introduce browser compatibility issues if just making the left/right values the same solves the problem.